projects/js-packages/components/components/icons/index.jsx

```diff
@@ -159,3 +261,11 @@ export function getIconBySlug( slug ) {
 
 	return iconsMap[ slug ];
 }
+
+export const SocialServiceIcon = ( { serviceName, className } ) => {
+	const Icon = getIconBySlug( serviceName );
+	if ( Icon ) {
+		return <Icon className={ className } />;
+	}
+	return null;
```

Comment:
```es6
return Icon ? <Icon className={ className } /> : null
```

Reply:
I think I just copied this over from the existing Product Icons code. I'll update this though.